client/Brocfile.js

```diff
@@ -1,8 +1,11 @@
 /* global require, module */
 
 var EmberApp = require('ember-cli/lib/broccoli/ember-app');
+var mergeTrees = require('broccoli-merge-trees');
+var pickFiles = require('broccoli-static-compiler');
 
-var app = new EmberApp({storeConfigInMeta: false});
```

Comment:
What's happening here? The `storeConfigInMeta` option is required for `ember-cli-rails`